frontend-ssr/pages/submissions.tsx

```diff
@@ -36,6 +42,27 @@ class Submissions extends React.Component<ConnectedProps<typeof connector>, Stat
     if (submissions.submissions === undefined) {
       this.getData();
     }
+
+    this.initIntersectionObserver = () => {
```

Comment:
Why are we dynamically creating this function instead of adding it on the class body?

Reply:
Also, this function should also start observing if we've not reached the end yet.